### DISCUSSION TITLE
Name for the config file

### DISCUSSION THREAD

**Discussion by Luiz-Micci** [thoughtworks/templify#27] [Ideas]
Today we are using template.yml as the plugin's configuration file and this will and will create confusion, as Backstage also uses a template.yml file for configuring its templates.
Therefore, we need to define a configuration file name.

Ideas:

- cookiecutter_config.yml
- plugin_replace.yml
- replace_mapping.yml
- cookiecutter_mapping.yml

**Comment by BeppeTW** [comment]
Moving away from Cookiecutter: maven-templify.yml.
